repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -231,12 +231,10 @@ object KeystoneRequestHandler {
   }
 
   def buildRetryValue(response: ServiceClientResponse): String = {
-    response.getHeaders.find(header => HttpHeaders.RETRY_AFTER.equalsIgnoreCase(header.getName)) match {
-      case Some(retryValue) => retryValue.getValue
-      case _ =>
-        val retryCalendar: Calendar = new GregorianCalendar
-        retryCalendar.add(Calendar.SECOND, 5)
-        DateUtils.formatDate(retryCalendar.getTime)
+    response.getHeaders(HttpHeaders.RETRY_AFTER).asScala.headOption.getOrElse {
```

Comment:
This is better, but it was safe previously.

Reply:
Yep. I was just making this consistent with the rest of the uses since I was already here.